polls/tests/test_admin.py

```diff
@@ -0,0 +1,122 @@
+            title='English',
+            code='en',
+            slug='english')
+        main.add_child(instance=self.english)
+        self.english.save_revision().publish()
+
+        # Create a site with the new homepage set as the root
+        Site.objects.all().delete()
+        Site.objects.create(
+            hostname='localhost', root_page=main, is_default_site=True)
+
+    def test_download_csv_question(self):
+        # make choices
+        choice1 = Choice(title='yes')
+        choice2 = Choice(title='no')
+        # make a question
+        question = Question(
+            title='is this a test',
+            allow_multiple_choice=True, show_results=False)
+        self.english.add_child(instance=question)
+        question.add_child(instance=choice1)
+        question.add_child(instance=choice2)
+        question.save_revision().publish()
+        # make a vote
+        client = Client()
+        client.login(username='tester', password='tester')
+
+        client.post(reverse('molo.polls:vote',
+                    kwargs={'question_id': question.id}),
+                    {'choice': [choice1.id, choice2.id]})
+        # should automatically create the poll vote
+        # test poll vote
+        response = download_as_csv(QuestionAdmin,
+                                   None,
+                                   Question.objects.all())
+        date = str(datetime.datetime.now().date())
+        expected_output = ('Content-Type: text/csv\r\nContent-Disposition:'
+                           ' attachment;filename=export.csv\r\n\r\n'
+                           'title,date_submitted,user,answer'
+                           '\r\nis this a test,'
+                           + date + ',tester,'
+                           '"yes,no"\r\n')
+        self.assertEquals(str(response), expected_output)
+
+    def test_download_csv_free_text_question(self):
+        question = FreeTextQuestion(
+            title='is this a test')
+        self.english.add_child(instance=question)
+        question.save_revision().publish()
+
+        client = Client()
+        client.login(username='tester', password='tester')
+        response = client.get('/')
+        self.assertContains(response, 'is this a test')
+
+        client.post(reverse('molo.polls:free_text_vote',
+                    kwargs={'question_id': question.id}),
+                    {'answer': 'this is an answer'})
+        response = download_as_csv(QuestionAdmin,
+                                   None,
+                                   Question.objects.all())
+        date = str(datetime.datetime.now().date())
+        expected_output = ('Content-Type: text/csv\r\nContent-Disposition:'
+                           ' attachment;filename=export.csv\r\n\r\n'
+                           'title,date_submitted,user,answer'
+                           '\r\nis this a test,'
+                           + date + ',tester,'
+                           'this is an answer\r\n')
```

Comment:
I think we'll need to quote the csv fields to handle cases where a field value has `,`s in it.